zenoh-flow/src/types.rs

```diff
@@ -202,6 +279,12 @@ pub fn default_output_rule(
     Ok(results)
 }
 
+/// The default input rules are a commodity function provided to
+/// users that do not need inputs rules in their operators.
+/// They provide a KPN input rule. Thus they trigger the execution only when
+/// all inputs are present.
+///
+/// It never returns an error.
```

Comment:
I would replace with:

Commodity function for users that need their operators to behave in a KPN manner: all inputs must be present before a computation can be triggered.